spec/core/ExpectationResultSpec.js

```diff
@@ -73,4 +73,30 @@ describe("buildExpectationResult", function() {
     var result = jasmineUnderTest.buildExpectationResult({actual: 'some-value'});
     expect(result.actual).toBe('some-value');
   });
+
+  it("handles nodejs assertions", function() {
+    var assert = require('assert');
```

Comment:
These specs also need to run in browsers, so we need to first check that we're in node.js where `require` exists before running this test.

Reply:
I'm not sure how to do it in Jasmine way. Do you have an example? Because I can check for `process`|`window`, `require` availability and so on.

Reply:
Take a look at [matchersUtilSpec](https://github.com/jasmine/jasmine/blob/7fb53dcdfa2dcf83682361a3cf160717be8a0d93/spec/core/matchers/matchersUtilSpec.js#L178) for an example of the opposite. The right thing to check for here might just be the presence of `require` with a `typeof` check.